testhelpers/flags.go

```diff
@@ -0,0 +1,33 @@
+package testhelpers
+
+import (
+	"flag"
+	"testing"
+)
+
+var integrationTest = flag.Bool("integration", false, "Run the integration go tests")
+var unitTest = flag.Bool("unit", false, "Run the unit go tests")
```

Comment:
Note to Reviewer(s):

We will need to pick a "default" set of tests to run. Currently (as @travisperson has pointed out in a comment) no tests are run unless the `-unit` or `-integration` flags are passed to them. Once we decided on a default set of tests we can set their flags default value to `true`.

Reply:
I like the idea of having all tests instrumented. It provides an easy way to find tests which are not instrumented, as we can pass false values to all flags and expect nothing to run :+1:

Reply:
The plan is to instrument all tests with these flags, the question I was getting at was: which flags should default to `true`?

Reply:
yep